app/_hub/tomkerkhove/microsoft_azure_container_instances/index.md

```diff
@@ -0,0 +1,222 @@
+---
+name: Microsoft Azure Container Instances
+publisher: Tom Kerkhove
+
+categories:
+  - deployment
+
+type: integration
+
+desc: Deploy Kong on Microsoft Azure Container Instances # (required) 1-liner description; max 80 chars
+# (required) extended description.
+  # Use YAML piple notation for extended entries.
+  # EXAMPLE long text format (do not use this entry)
+  # description: |
+  #   Maintain an indentation of two (2) spaces after denoting a block with
+  #   YAML pipe notation.
+  #
+  #   Lorem Ipsum is simply dummy text of the printing and typesetting
+  #   industry. Lorem Ipsum has been the industry's standard dummy text ever
+  #   since the 1500s.
+description: |
+  This guide walks you through deploying and running Kong on Microsoft Azure Container Instances.
+
+  We will be using either Azure Database for PostgreSQL or Cosmos Db to store the gateway configuration.
+support_url: # (Optional) A specific URL of your own for this extension.
```

Comment:
Whoops, this isn't actually optional - I need to update the template - it should be https://github.com/tomkerkhove/kong-deployment-on-azure/issues I think

Reply:
Good point :+1: